### ISSUE TITLE
"Remember password? Sign in" link may not work?

### ISSUE THREAD

**Issue by rfk** [mozilla/fxa-auth-mailer#203]
Our password-reset email contains a "Remember password? Sign in" link:

![screen shot 2016-08-31 at 11 30 11](https://cloud.githubusercontent.com/assets/34695/18112845/52825a9c-6f6e-11e6-9f9a-666b471223da.png)

With a url like the following:

```
https://accounts.firefox.com/signin?email=rfkelly%40mozilla.com&utm_source=email&utm_medium=email&utm_campaign=fx-forgot-password&utm_content=fx-remember-password
```

It's not obvious to me whether this link will do anything useful.  Following it will let you sign in to Firefox Accounts on the web, but since it doesn't contain e.g. `context=fx_desktop_v3` or similar, I don't think it will enable you to actually sign in to sync.  And if we can't cause the user to be properly signed in, perhaps we shouldn't have this link at all.

@shane-tomlinson @ryanfeeley thoughts?

**Comment by shane-tomlinson** [comment]
Good point. Two things are required for the user to successfully sign in:

1) A resume token on the signin link. The resume token will contain the context and other information required to complete the flow.
2) Even with the resume token, the user must sign in from the same browser they initiated the reset password from.

**Comment by ryanfeeley** [comment]
@rfk Well spotted. Perhaps the most correct thing to do is to remove the line.

**Comment by ryanfeeley** [comment]
Maybe a better question is, should we be including a way to activate Sync when users are signed into FxA Settings in Firefox (but not signed in to Sync).

One of the things I am considering is moving sync datatype selection entirely to the web (and have it read-only in the client). Gotta get back to my sprint work now :(

**Comment by rfk** [comment]
> should we be including a way to activate Sync when users are signed into FxA Settings in Firefox (but not signed in to Sync).

IIRC you've specced exactly this behavior in the devices view [1], or is this something slightly different?

[1] https://github.com/mozilla/fxa/blob/fxa-89-devices-view/features/FxA-89-devices-view/README.md

**Comment by rfk** [comment]
> Perhaps the most correct thing to do is to remove the line.

Agreed, if we know it's not going to work then there's no point in having it there.
